rdt/hyper_transformer.py

```diff
@@ -162,15 +162,44 @@ def _unfit(self):
         self._fitted_fields.clear()
         self._fitted = False
 
-    def get_field_data_types(self):
-        """Get the ``field_data_types`` dict.
+    @staticmethod
+    def _validate_config(config):
+        sdtypes = config.get('sdtypes', {})
+        transformers = config.get('transformers', {})
+        for column, transformer in transformers.items():
+            input_type = transformer.get_input_type()
+            sdtype = sdtypes.get(column)
+            if input_type != sdtype:
+                warnings.warn(f'You are assigning a {input_type} transformer to a {sdtype} column'
```

Comment:
Should we then change all the `prints` that `warn` to warnings ? I'm getting a little bit confused with `print` and `warning`.